OP-875 link bill admission - doc (#177)
informatici/openhospital-doc: doc_user/UserManual.adoc

To modify an existing bill select the bill in the *_Patient Bills Management_* window and then press the *[.underline]##E##dit Bill* button. The *_Edit Patient Bill_* window is shown:

image:image61a.png[Edit Patient Bill Female Ward]

> NOTE: Only bills with status “O” (Open) can be modified by enabled users (See <<users-groups, User & Groups>>), otherwise, an A4 report will be shown.

Reviewer: ```suggestion
NOTE: Only bills with a status of “O” (Open) can be modified by enabled users (See <<users-groups, User & Groups>>), otherwise, an A4 report is shown.
```

> If the bill, at the time of creation, was related to an admitted patient, the related ward will be shown at the upper right. 

Reviewer: ```suggestion
If the bill, at the time of creation, was related to an admitted patient, the related ward is shown in the upper right. 
```

> If the patient's status has changed since then, the application will ask what to do:

Reviewer: ```suggestion
If the patient's status has changed the application prompts for the next action:
```

*Case 1)*

image:image61b.png[Question This patient is no longer admitted. Do you want to unlink this bill from the previous admission?]

> If the answer selected is "Yes", the bill will be updated and will belong under the "OPD" category (no admission).

Reviewer: ```suggestion
If the answer selected is "Yes", the bill is updated and belongs to the "OPD" category (no admission).
```

image:image61c.png[Edit Patient Bill OPD]

> If the answer selected is "No", the bill remains unchanged and will belong to the original admission.

Reviewer: ```suggestion
If the answer selected is "No", the bill remains unchanged and continues to belong to the original admission.
```

image:image61a.png[Edit Patient Bill Female Ward]

*Case 2)*

image:image61d.png[Question This patient is admitted now. Do you want to link this bill to the current admission?]

> If the answer selected is "Yes", the bill will be updated and will belong to the current admission.

Reviewer: ```suggestion
If the answer selected is "Yes", the bill is updated and belongs to the current admission.
```

image:image61e.png[Edit Patient Bill Maternity Ward]

> If the answer selected is "No", the bill remains unchanged and will belong under the "OPD" category (no admission)

Reviewer: ```suggestion
If the answer selected is "No", the bill is unchanged and is associated with  the "OPD" category (no admission)
```

image:image61c.png[Edit Patient Bill OPD]

*Case 3)*

image:image61f.png[Question This bill was linked to a previous admission. Do you want to link it to the current admission instead?]

> If the answer selected is "Yes", the bill will be updated and will belong to the current admission.

Reviewer: ```suggestion
If the answer selected is "Yes", the bill is updated and belongs to the current admission.
```

image:image61e.png[Edit Patient Bill Maternity Ward]

> If the answer selected is "No", the bill remains unchanged and will belong to the original admission.

Reviewer: ```suggestion
If the answer selected is "No", the bill is unchanged and belongs to the original admission.
```

image:image61a.png[Edit Patient Bill Female Ward]

> Bill status can be easily identified in the last column in "Patient Bills Management" windows:

Reviewer: ```suggestion
The status of a bill is found in the last column in "Patient Bills Management" windows:
```

* *Second Row*: the *PAID* and *UNPAID* for the specified *Period*
* *Third Row*: the *PAID* and *UNPAID* for the current *user* for *Today*

> The Administrator, or any enabled user (See <<users-groups, User & Groups>>) can filter all incomes for all users have some payment amount:

Reviewer: ```suggestion
The Administrator, or any enabled user (See <<users-groups, User & Groups>>), can filter all incomes for all users with a  payment amount:
```
